# request
string model_name
string model_sdf_file
float32 pos_x
float32 pos_y
float32 pos_z
float32 rot_w
float32 rot_x
float32 rot_y
float32 rot_z
---
# response
bool success
